apps/credential-showcase-api-server/src/__tests__/asset.repository.test.ts

```diff
@@ -1,23 +1,36 @@
+import 'reflect-metadata';
+import { PgDatabase } from 'drizzle-orm/pg-core';
 import { drizzle } from 'drizzle-orm/pglite'
 import { migrate } from 'drizzle-orm/node-postgres/migrator';
-import AssetRepository from '../database/repositories/assetRepository';
+import { Container } from 'typedi';
+import { DatabaseService } from '../services/DatabaseService';
+import AssetRepository from '../database/repositories/AssetRepository';
 import { NewAsset } from '../types';
 
 describe('Database asset repository tests', (): void => {
     let repository: AssetRepository;
 
     beforeEach(async (): Promise<void> => {
-        const database: any = drizzle();
+        const database: PgDatabase<any> = drizzle();
```

Comment:
```suggestion
        const database = drizzle() as unknown as NodePgDatabase<Record<string, never>>;
```
Let's atleast cast it as NodePG, once for some reason the the proper typeing is not working 
```
import { PGlite } from '@electric-sql/pglite';
import { drizzle, type PgliteDatabase } from 'drizzle-orm/pglite';

const ENABLE_LOGGING = false;

let db: PgliteDatabase;
let client: PGlite;

beforeAll(async () => {
	client = new PGlite();
	db = drizzle(client, { logger: ENABLE_LOGGING })
});

afterAll(async () => {
	await client?.close();
});
```
reference 
https://github.com/drizzle-team/drizzle-orm/blob/e5c63db0df0eaff5cae8321d97a77e5b47c5800d/integration-tests/js-tests/driver-init/module/pglite.test.mjs#L9